.github/workflows/generate-initial-checksums.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         env:
           BASE_EXPERIMENT_LOCATION: ${{ env.EXPERIMENT_LOCATION }}/base-experiment
         run: |
-          ssh -i ${{ steps.ssh.outputs.private-key-path }} /bin/bash<<EOT
+          ssh ${{ secrets.SSH_USER }}@${{ secrets.SSH_HOST }} -i ${{ steps.ssh.outputs.private-key-path }} /bin/bash<<EOT
```

Comment:
`EOT` isn't quoted. No idea if it is important or not.

Reply:
For heredocs:
Quoted `'EOT'` means variable and command substitution within the here document is prevented.
Unquoted `EOT` means variables or commands inside are automatically replaced with their respective values.
Quoting or not quoting doesn't matter in this instance because we don't have any env vars or anything